cypress/support/commands.ts

```diff
@@ -1048,6 +1048,14 @@ export const setFeatureFlags = (flags: FlagMap): Cypress.Chainable => {
   })
 }
 
+export const setFeatureFlagsNoNav = (flags: FlagMap): Cypress.Chainable => {
```

Comment:
The reason `setFeatureFlags` wasn't working on the checkout page is that `TreeNav` doesn't display there; so we were failing the 'wait for tree nav' check that is implicit in `setFeatureFlags`. 

This adds a separate cypress command we can use to override feature flags on pages that don't have `TreeNav`.